vuepress/docs/next/tutorials/getting-started/openshift-install.md

```diff
@@ -124,8 +97,6 @@ If deploying your Entando Application into your OpenShift namespace generates pe
 Check with your cluster administrator if you need help assigning Roles. Generally, this requires the creation of a Role, preferably a ClusterRole with the above permissions. Your Entando installer needs to be given this Role in the target namespace, in accordance with how administrators manage security. For example, if the ClusterRole we create is `entando-installer` and the user's name is John, to create rolebinding on OpenShift use:
```

Comment:
link "above permissions" to where these are on the page?

we create -> you create, unless Entando is creating the role for the user